R/istr.R

```diff
@@ -41,9 +41,7 @@ istr <- function(companies, scenarios, inputs) {
 #' @export
 istr_at_product_level <- function(companies,
                                   scenarios,
-                                  inputs,
-                                  low_threshold = 1 / 3,
-                                  high_threshold = 2 / 3) {
```

Comment:
Removing these arguments makes the code less flexible. The user no longer can customize them. This also make it harder to test and thus maintain.